From 43ac58f2a867e8a4911e7297bc0a04bed0b8163e Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Mon, 28 Mar 2022 15:56:42 +0800
Subject: [PATCH] remove function for set uboot enviroment

fullmetalupdate os rollback feature make use of the uboot enviroment
variable. while wrlinux ostree don't use uboot enviroment variable.
so remove this part of function in case of error message in log.

Upstream-Status: Pending

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 fullmetalupdate/fullmetalupdate_ddi_client.py | 1 -
 fullmetalupdate/updater.py                    | 2 --
 2 files changed, 3 deletions(-)

diff --git a/fullmetalupdate/fullmetalupdate_ddi_client.py b/fullmetalupdate/fullmetalupdate_ddi_client.py
index 0ac9f01..e2cd41c 100755
--- a/fullmetalupdate/fullmetalupdate_ddi_client.py
+++ b/fullmetalupdate/fullmetalupdate_ddi_client.py
@@ -350,7 +350,6 @@ class FullMetalUpdateDDIClient(AsyncUpdater):
         try:
             self.pull_ostree_ref(False, rev_number)
             self.ostree_stage_tree(rev_number)
-            self.delete_init_var()
         except Exception as e:
             self.logger.error("Updating the OS failed ({})".format(e))
             return False
diff --git a/fullmetalupdate/updater.py b/fullmetalupdate/updater.py
index 42f5cea..a6f012f 100755
--- a/fullmetalupdate/updater.py
+++ b/fullmetalupdate/updater.py
@@ -46,8 +46,6 @@ class AsyncUpdater(object):
 
         self.logger = logging.getLogger('fullmetalupdate_container_updater')
 
-        self.mark_os_successful()
-
         bus = SystemBus()
         self.systemd = bus.get('.systemd1')
 
-- 
2.25.1

